ctwrap/simulation.py

```diff
@@ -154,10 +153,10 @@ def _save(self, mode="a", task=None, **kwargs: str) -> None:
         force = self._output['force_overwrite']
 
         if filepath is not None:
-            filename = os.path.join(filepath, filename)
+            filename = PurePath.joinpath(Path(filepath), filename)
```

Comment:
`PurePath.joinpath(Path(filepath), filename)`

can be replaced by 

`Path(filepath) / filename`

(here and elsewhere)